src/main/java/com/eva/model/person/staff/Comments.java

```diff
@@ -0,0 +1,47 @@
+package com.eva.model.person.staff;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Comments {
```

Comment:
Do you want to change the name of Comments to something like CommentList so that it is more clear that this is a list of comments